crypto/keys/keyring.go

```diff
@@ -370,7 +370,7 @@ func (kb keyringKeybase) ImportPubKey(name string, armor string) error {
 // deleting it (for security). It returns an error if the key doesn't exist or
 // passphrases don't match. The passphrase is ignored when deleting references to
 // offline and Ledger / HW wallet keys.
-func (kb keyringKeybase) Delete(name, passphrase string, skipPass bool) error {
+func (kb keyringKeybase) Delete(name, _ string, _ bool) error {
```

Comment:
should we remove the params if they're not being used?

Reply:
I honestly feel mixed about modifying the interface at this stage. My gut feeling suggests me to avoid at least breaking the `Keybase` for now. I don't have a strong opinion though.

Reply:
let's see what the others might say about this. I don't have a strong preference either